packages/core/src/App/Components/Routes/binary-link.jsx

```diff
@@ -11,13 +10,13 @@ import { observer, useStore } from '@deriv/stores';
 // when binary link is imported into components present in routes config
 // or into their descendants
 const BinaryLink = observer(({ active_class, to, children, href, ...props }) => {
-    const platform_context = React.useContext(PlatformContext);
-    const { common } = useStore();
+    const { common, client } = useStore();
     const { has_error, error } = common;
     const setError = error.setError;
-    const is_appstore = platform_context?.is_appstore;
     const path = normalizePath(to);
-    const route = findRouteByPath(path, getRoutesConfig({ is_appstore }));
+    const { is_eu } = client;
+    const is_eu_country = is_eu;
+    const route = findRouteByPath(path, getRoutesConfig({ is_eu_country }));
```

Comment:
Can you explain this code, why are you passing `is_eu_country` inside `getRoutesConfig`?

Reply:
As discussed `is_eu_country` was being used inside the `getRoutesConfig` but now we have removed. 
Thanks @matin-deriv for highlighting